content/projects/metabake.md

```diff
@@ -0,0 +1,46 @@
+---
+title: MetaBake.org
+repo: metabake/_mBake
+homepage: https://metabake.org
+language:
+  - Pug
+license:
+  - GNU GPL v3.0
```

Comment:
Please change to "GPL v3.0" (that's just what we're using for that license in the list).